src/calendar/day/period/index.js

```diff
@@ -129,7 +129,8 @@ class Day extends Component {
 
     if (this.props.marking) {
       containerStyle.push({
-        borderRadius: 17
+        borderRadius: 17,
+	backgroundColor: this.markingStyle.day && this.markingStyle.day.color ? this.markingStyle.day.color : this.theme.calendarBackground
```

Comment:
The containerStyle's backgroundColor is treated a few rows ahead. I don't see why you add it here.